IPython/core/interactiveshell.py

```diff
@@ -924,7 +924,9 @@ def init_virtualenv(self):
         
         # In Cygwin paths like "c:\..." and '\cygdrive\c\...' are possible
         if str(p_venv).startswith("\\cygdrive"):
```

Comment:
I've just noticed this, but not sure if it's correct (haven't checked)

```suggestion
        if p_venv.parts[1] == cygdrive:
```

Reply:
This is neat! Just tested it works.